### PR TITLE
KAZOO-4854: try to connect default fs node if none configured

### PR DESCRIPTION
**Pull request by jamhed**

### CODE REVIEW

File: applications/ecallmgr/src/ecallmgr_fs_nodes.erl [hunk 1]
```diff
@@ -760,25 +760,47 @@ start_node_stats(#node{}) ->
...
+                     'success';
+                 'undefined' ->
+                     lager:debug("failed to receive a response for node configs");
+                 _E ->
+                     lager:debug("received a non-list for fs_nodes: ~p", [_E])
+             end,
+    case Result of
+        'success' -> 'ok';
+        _ ->
+            _ = maybe_add_default_fs(),
```
Comment (k-anderson) [inline applications/ecallmgr/src/ecallmgr_fs_nodes.erl]: @jamhed if you are able to connect to the default then don't enter the polling stage

File: applications/ecallmgr/src/ecallmgr_fs_nodes.erl [hunk 2]
```diff
@@ -760,25 +760,47 @@ start_node_stats(#node{}) ->
...
+-spec maybe_add_default_fs() -> 'skip' | 'ok' | {'error', 'no_connection'}.
+maybe_add_default_fs() ->
+    Node = default_fs_node(),
+    lager:info("attempting to connect default freeswitch node ~p", [Node]),
+    case net_adm:ping(Node) of
+        'pong' -> add(Node);
+        _ -> 'skip'
+    end.
+
 start_node_from_config(MaybeJObj) ->
```
Comment (k-anderson) [inline applications/ecallmgr/src/ecallmgr_fs_nodes.erl]: @jamhed I know that its not your function, but if you seeing something without a spec please correct it ;)

File: applications/ecallmgr/src/ecallmgr_fs_nodes.erl (hunk 1 above)
Comment (jamhed) [inline applications/ecallmgr/src/ecallmgr_fs_nodes.erl]: @k-anderson what do you mean? the logic here is that first we try a configuration parameter and if it fails for any reason then we try to connect to default node.

File: applications/ecallmgr/src/ecallmgr_fs_nodes.erl (hunk 2 above)
Comment (jamhed) [inline applications/ecallmgr/src/ecallmgr_fs_nodes.erl]: @k-anderson ok

File: applications/ecallmgr/src/ecallmgr_fs_nodes.erl (hunk 1 above)
Comment (k-anderson) [inline applications/ecallmgr/src/ecallmgr_fs_nodes.erl]: @jamhed Howdy, I meant something like:

case maybe_add_default_fs() of
    'ok' -> 'ok';
    _Else ->
             timer:sleep(5 * ?MILLISECONDS_IN_SECOND),
              start_preconfigured_servers()
end.

**Pull request merged**